apps/src/redux.js

```diff
@@ -1,21 +1,124 @@
+/* eslint no-unused-vars: "error" */
 /** @file Code.org configured store-creation method.
  *  @see http://redux.js.org/docs/api/createStore.html */
 import Immutable from 'immutable';
 import experiments from "./util/experiments";
+import * as redux from 'redux';
+import reduxThunk from 'redux-thunk';
+import { connect } from 'react-redux';
 
-var redux = require('redux');
-var reduxThunk = require('redux-thunk').default;
 if (process.env.NODE_ENV !== "production") {
   var createLogger = require('redux-logger');
 }
 
+
+let reduxStore;
+let globalReducers = {};
+
+let __oldReduxStore;
+let __oldGlobalReducers;
+
+export function stubRedux() {
+  if (__oldReduxStore) {
+    throw new Error("Reduce store has already been stubbed. Did you forget to call restore?");
+  }
+  __oldReduxStore = reduxStore;
+  __oldGlobalReducers = globalReducers;
+  reduxStore = null;
+  globalReducers = {};
+}
+
+export function restoreRedux() {
+  reduxStore = __oldReduxStore;
+  globalReducers = __oldGlobalReducers;
+  __oldReduxStore = null;
+  __oldGlobalReducers = null;
+}
```

Comment:
Feels weird to have stubbing/restoring here. Could this live in a test file instead?

Also, I would argue that `stubRedux` is a bit of a misnomer, as we're not really stubbing redux here - we're backing up and clearing it.